### DISCUSSION TITLE
Welcome to KitQL Discussions!

### DISCUSSION THREAD

**Discussion by jycouet** [jycouet/kitql#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with `KitQL`.

**Comment by draylegend** [comment]
I'm an angular dev with there years of experience. Love nrwl/nx and the monorepo approach.

Currently looking around for slimm full stack frameworks with graphql or http2 stream support.

I like the way svelte solves the bundle size! Like the idea of full stack framework. And the best part for me of sveltekit, as an rxjs user, is that I don't need to unsubscribe in most cases 🥳

Happy coding 🔥🚀
